src/lib.rs

```diff
@@ -149,7 +154,15 @@ impl<Twi: I2CForT1, D: DelayUs<u32>> Se050Backend<Twi, D> {
     }
 }
 
-const REQUIRED_CURVES: &[CurveInitializer] = &[PRIME256V1_INITIALIZER, SECP521R1_INITIALIZER];
+const REQUIRED_CURVES: &[CurveInitializer] = &[
+    PRIME256V1_INITIALIZER,
+    SECP384R1_INITIALIZER,
+    SECP521R1_INITIALIZER,
+    BRAINPOOL_P256R1_INITIALIZER,
+    BRAINPOOL_P384R1_INITIALIZER,
+    #[cfg(feature = "large-ecc")]
+    BRAINPOOL_P512R1_INITIALIZER,
```

Comment:
Doesn’t this mean that configure version 2 can have different meanings depending on the activated features (which may change between firmware versions)?  We could always initialize it to be sure.